code/modules/antagonists/fugitive/old_god.dm

```diff
@@ -0,0 +1,119 @@
+
+/mob/camera/yalp_elor
```

Comment:
You might want to add a way for Yalp to quickly jump to the locations of its cultists, in case it gets lost or something.

Reply:
You may want to add a custom "Pray" for the cultists that create one of those alerts that ghosts can click and go to location, but for Yalp to use.

Reply:
I'll do ATH1909's idea, easy fix